### ISSUE TITLE
Azure CI build errors

### ISSUE THREAD

**Issue by mitza-oci** [DOCGroup/XSC#46]
Looks like using the distro's Boost may not be compatible with `-std=c++11`

**Comment by jwillemsen** [comment]
It worked in the past when I did setup the XSC builds on azure, didn't got an error email from Azure at the moment it started failing :-(

**Comment by mitza-oci** [comment]
Yeah, it's been difficult to follow the changes they're making to the base images.  

In general I think it's better to be decoupled from the distro packagers for things that are C++-heavy like Qt, Boost, and Xerces.  Doesn't matter as much for OpenSSL since it's all C.

**Comment by jwillemsen** [comment]
XSC has a requirement on boost, maybe you can find a fix for the CI builds?

**Comment by jwillemsen** [comment]
Disabling c++11 doesn't help, see #48

**Comment by mitza-oci** [comment]
> XSC has a requirement on boost, maybe you can find a fix for the CI builds?

What I've done in the past for other projects was downloaded and built from source.  This was for Travis but it should work the same for Azure's Linux VM.

```
export LD_LIBRARY_PATH=$TRAVIS_BUILD_DIR/ext/lib
wget https://dl.bintray.com/boostorg/release/1.64.0/source/boost_1_64_0.tar.bz2
tar xjf boost_1_64_0.tar.bz2
cd boost_1_64_0
./bootstrap.sh --prefix=$TRAVIS_BUILD_DIR/ext
echo 'using gcc : 7 : g++-7 ;' >> project-config.jam
BOOST_TOOLSET=gcc-7
BOOST_LIBRARIES="chrono context coroutine date_time filesystem locale program_options serialization signals system test thread"
for libname in $BOOST_LIBRARIES; do boost_libs+=("--with-$libname"); done
./b2 -d0 -j4 ${boost_libs[@]} toolset=$BOOST_TOOLSET link=static install
```

**Comment by jwillemsen** [comment]
Maybe vcpkg also works

**Comment by jwillemsen** [comment]
Could you try your script through a new PR?

**Comment by jwillemsen** [comment]
Fixed by #49, thanks for your script suggestion
